README.md

```diff
@@ -0,0 +1,31 @@
+# LaTiS Server Project Template
+
+A [Giter8][g8] template for creating runnable LaTiS servers.
```

Comment:
I don't know about the wording here. I also used similar wording in the description of this repository and the `description` in `default.properties`.